docs/resources/dw_vw_impala.md

```diff
@@ -0,0 +1,44 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "cdp_dw_vw_impala Resource - terraform-provider-cdp"
+subcategory: ""
+description: |-
+  
+---
+
+# cdp_dw_vw_impala (Resource)
```

Comment:
You will need an example file too: examples/resources/cdp_dw_vw_impala/resource.tf

Once you add it, render the docs again, that will add the example to the doc

Reply:
Sure, added and generated docs again.